### ISSUE TITLE
[XamarinProfiler-MAC]:Accessibility: Voiceover: MAS4.2.1: The name property of the triangular buttons (i.e., Options, Statistics, Samples per thread buttons) are not being announced by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#439]
### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations/Time Profiler" Profiling page.
4. Navigate to the any of the below elements:
a. Allocations-> Allocations/Call Tree tab-> Inspect Tab-> Triangular Buttons (i.e., Options, statistics, Top allocations)
b. Time Profiler-> Call Tree tab-> Inspect Tab-> Triangular Buttons (i.e., Options, statistics, Top allocations)
5. Observe the VoiceOver behavior for these triangular  expand/collapse button.

### Expected Behavior
The VoiceOver Should announce the concerned name of the triangular buttons like Options triangular expanded/collapsed  button, Statistics triangular expanded/collapsed button etc.
### Actual Behavior
The VoiceOver is announcing as "Expanded, Disclosure triangle", but its not announcing the name of the purpose of the triangular button.
This issue is observed for all the triangular buttons available in the this inspect tabs.
### User Impact
The User won't be able to know the purpose of the button until its name is not being announced
### Screenshots
![voiceover_profilermac4](https://user-images.githubusercontent.com/22956130/41716285-91919baa-7573-11e8-8136-701f5c36f88f.png)
![voiceover_profilermac4 2](https://user-images.githubusercontent.com/22956130/41716299-98b21dec-7573-11e8-82c5-40255cdb6b44.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400
